modules/optimization/examples/diffusion_reaction_XYDelaunay/tests

```diff
@@ -0,0 +1,27 @@
+[Tests]
```

Comment:
I dont see what is the point of this test. 
How is XYDelauany any special here?

Reply:
We were exploring the differences when measurement points were on a node or off. We wanted to document a case for optimization where you can create a geometry with nodes at specified locations easily.